src/fparser/Fortran2003.py

```diff
@@ -156,20 +158,23 @@ class Base(ComparableMixin):
                 is either str or FortranReaderBase.
       .item   - Line instance (holds label) or None.
     """
+    # This dict of subclasses is populated dynamically by code at the end
+    # of this module. That code uses the entries in the
+    # 'subclass_names' list belonging to each class defined in this module.
     subclasses = {}
 
     @show_result
     def __new__(cls, string, parent_cls=None):
         """
         Create a new instance of this object.
 
-        :param cls: the class of object to create
-        :type cls: :py:type:`type`
+        :param type cls: the class of object to create
         :param string: (source of) Fortran string to parse
         :type string: str or :py:class:`FortranReaderBase`
         :param parent_cls: the parent class of this object
         :type parent_cls: :py:type:`type`
         """
+        from . import readfortran
```

Comment:
I thought relative imports were bad?

Reply:
I'm unsure :-) Anyway, I've made it explicit.